### DISCUSSION TITLE
Assistance Needed: Login Issue Resolution | | Hobby

### DISCUSSION THREAD

**Discussion by umair-tauni** [vercel/community#6306] [Account Issue]
### Question

Case number # 00213978

I am writing to seek assistance regarding an issue I encountered while
attempting to access my account. Upon creating an account using the email
address,I successfully completed the email
verification process. However, upon attempting to log in, I consistently
encounter a pop-up error indicating "authentication failed."
In an effort to resolve this matter, I attempted to utilize the GitHub
login option. However, I received a notification stating that the email
address is already associated with an existing account.

Could you kindly assist me in resolving this login issue at your earliest
convenience? Your prompt attention to this matter would be greatly
appreciated.

![Screenshot from 2024-03-28 12-48-23](https://github.com/vercel/community/assets/155439061/cd970e26-8ac4-4552-9e0e-6f8a6137de47)

![Screenshot from 2024-03-28 12-46-01](https://github.com/vercel/community/assets/155439061/662e67d1-074f-43e0-a565-ef103c0c84e2)

**Comment by amyegan** [comment]
Hey @umair-tauni. ~~Please reach out to our account specialists at [registration@vercel.com](mailto:registration@vercel.com).~~

https://vercel.com/guides/why-can-i-not-signup

**Reply by amyegan** [reply]
I just noticed that you already have a case number, so you've already done exactly what you need to do to solve this issue. You should expect one of our specialists to respond as soon as they are available.

**Reply by umair-tauni** [reply]
Thank you @amyegan.. Hope it will be resolved soon.
